compatibility/bazel_tools/testing.bzl

```diff
@@ -150,6 +152,17 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "start": "0.0.0",  #fix to first snapshot after now
```

Comment:
I assume this still needs to be changed before the merge?

Reply:
The way I understand the protocol for compatibility testing is that  these values will be filled in as part of the release. I.e. this value would become the first release that includes the `ParticipantPruningIT` test suite. Is that right @moritzkiefer-da ?

Reply:
Yeah it’s a bit annoying, we should add exclusive bounds but atm you’re stuck with merging this and then fixing it up on the next snapshot.